app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreen.kt

```diff
@@ -84,9 +85,10 @@ fun AddEditTaskScreen(
         )
 
         // Check if the task is saved and call onTaskUpdate event
-        LaunchedEffect(uiState.isTaskSaved) {
-            if (uiState.isTaskSaved) {
-                onTaskUpdate()
+        if (uiState.isTaskSaved) {
```

Comment:
Would this run on every recomposition, regardless of if `uiState.isTaskSaved` changed or not?

Reply:
Maybe this is answered here: https://stackoverflow.com/questions/69085027/difference-between-remember-and-rememberupdatedstate-in-jetpack-compose

Reply:
Yeah. This if statement will be check on every recomposition